charts/catalog/templates/controller-manager-deployment.yaml

```diff
@@ -77,7 +77,7 @@ spec:
             port: 8080
             path: /healthz
           failureThreshold: 1
-          initialDelaySeconds: 10
+          initialDelaySeconds: 20
```

Comment:
did you mean to commit this?

Reply:
If so might be useful as a separate PR which could merge faster than this one :)